### PR TITLE
remove extra links

### PR DESCRIPTION
**Pull request by BillWagner**
additional feedback.

### CODE REVIEW

File: docs/index.yml [hunk 1]
```diff
@@ -57,13 +57,6 @@ conceptualContent:
         - url: core/tutorials/index.md
```
Comment (mairaw) [inline docs/index.yml]: line 53 - can you change from Deploying to Deploy?

File: docs/index.yml [hunk 2]
```diff
@@ -57,13 +57,6 @@ conceptualContent:
...
           itemType: tutorial
           text: Tutorials
-        - url: core/porting/index.md
-          itemType: how-to-guide
-          text: Port apps from .NET Framework
-      # footerLink (optional)
-      footerLink:
-        url: core/index.md
```
Comment (mairaw) [inline docs/index.yml]: wondering if we should keep this one and the next

**Review by mairaw — APPROVED** [review]
LGTM. Left a few comments for you to consider.

File: docs/index.yml (hunk 2 above)
Comment (BillWagner) [reply docs/index.yml]: I did a bit of rearranging, and kept these.

**Pull request merged**